tests/test_routes.py

```diff
@@ -0,0 +1,46 @@
+from app.models.planet import Planet
+
+def test_get_all_planets_when_none_exist(client):
+    response = client.get("/planets")
+
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_planet_1_when_none_exist(client):
+    response = client.get("/planets/1")
+
+    response_body = response.get_json()
+
+    assert response.status_code == 404
+    assert response_body == "Error: Planet 1 not found"
+
+def test_get_all_planets_when_some_exist(client, two_saved_planets):
+    planet1 = dict(id=1, name="Planet 1", description="I'm planet 1", type="gas giant")
+    planet2 = dict(id=2, name="Planet 2", description="I'm planet 2", type="gas giant")
```

Comment:
If `two_saved_planets` actually returned the records it created in the fixture, that list would be available here and we wouldn't need to build the dictionaries ourselves. That being said, sometimes it's nice to explicitly build the results we are looking for so that we know what's being tested simply by looking at the test itself. It can be a bit of a balancing game.